hack/helmfile.yaml

```diff
@@ -129,6 +129,12 @@ releases:
         value: {{ .Values.federationDomain1 }}
       - name: brig.config.optSettings.setFederationDomainConfigs[0].domain
         value: {{ .Values.federationDomain2 }}
+      - name: brig.config.optSettings.setFederationDomainConfigs[2].domain
+        value: {{ .Values.dynBackendDomain1 }}
+      - name: brig.config.optSettings.setFederationDomainConfigs[3].domain
+        value: {{ .Values.dynBackendDomain2 }}
+      - name: brig.config.optSettings.setFederationDomainConfigs[4].domain
+        value: {{ .Values.dynBackendDomain3 }}
```

Comment:
I don't understand this, but two lines above the index was 0, and this block starts with 2. Is index 1 skipped on purpose?